Tools/autotest/sim_vehicle.py

```diff
@@ -216,6 +216,18 @@ def find_root_dir():
     """Return path to root directory"""
     return os.path.realpath(os.path.join(find_autotest_dir(), '../..'))
 
+
+def wait_unlimited():
```

Comment:
Why not `time.sleep(987654321987654321)`?  I'll fix any user bug report that comes in because that timer expired.